src/smtsolvers/LookaheadSMTSolver.cc

```diff
@@ -1,16 +1,59 @@
-//
-// Created by prova on 07.02.19.
-//
+/*
+ * Copyright (c) 2022, Antti Hyvarinen <antti.hyvarinen@gmail.com>
+ * Copyright (c) 2022, Konstantin Britikov <konstantin.britikov@usi.ch>
+ *
+ * SPDX-License-Identifier: MIT
+ */
 
 #include "LookaheadSMTSolver.h"
+#include "Proof.h"
+
+
+void LookaheadSMTSolver::attachClause(CRef cr) {
+    const Clause& c = ca[cr];
```

Comment:
```suggestion
    Clause const & c = ca[cr];
```
Is the current style, I believe. What about that linter? :)